cfme/tests/configure/test_rest_access_control.py

```diff
@@ -509,12 +504,11 @@ def test_create_uppercase_user(self, request, appliance):
             "name": "REST User {}".format(uniq),
             "password": fauxfactory.gen_alphanumeric(),
             "email": "user@example.com",
-            "group": {"description": "EvmGroup-user_self_service"}
```

Comment:
same here if you do not require exactly ```"user@example.com"```.

Reply:
Here maybe we can use gen_email.